tests/docker-compose.yml

```diff
@@ -7,7 +7,8 @@ services:
       context: .
       dockerfile: test_pipenv.Dockerfile
       args:
-        PIPENV_VERSION: "2018.5.18"
+        PIPENV_VERSION: "2020.8.13"
+        VIRTUALENV_VERSION: "20.0.33"
```

Comment:
Did you want to update this to 20.4.5 as well?

Reply:
nope - trying to use different versions in the test than the default.